### PR TITLE
feat(proto): Refactor app to support proto v2

### PR DESCRIPTION
**Pull request by askolesov**

### CODE REVIEW

File: x/cheqd/types/diddoc_verification_method.go [hunk 1]
```diff
@@ -67,22 +55,29 @@ func VerifySignature(vm VerificationMethod, message []byte, signature []byte) er
 	var verificationError error
 
 	switch vm.Type {
-	case Ed25519VerificationKey2020:
-		_, keyBytes, err := multibase.Decode(vm.PublicKeyMultibase)
+	case Ed25519VerificationKey2020{}.Type():
+		var ed25519VerificationKey2020 Ed25519VerificationKey2020
```
Comment (askolesov) [inline x/cheqd/types/diddoc_verification_method.go:59]: Validation and signature verification should belong to verification materials.

**Comment by askolesov** [comment]
It's ready for review. Two items to mention:
* Proto breaking changes detection is temporarily disabled;
* Validation and signature verification should be moved to corresponding verification material structures. This is small change, will do it later.

**Review by anikitinDSR — DISMISSED** [review]

**Review by Eengineer1 — APPROVED** [review]
Looks great, thanks!
There's still some pending confusion with renaming the `cheqd` module to `did`, as I see for now.

File: x/cheqd/types/diddoc_verification_method.go (hunk 1 above)
Comment (Eengineer1) [reply x/cheqd/types/diddoc_verification_method.go]: @askolesov Makes sense.

**Comment by askolesov** [comment]
As for renaming, I'm going to rename:
* Go package;
* Go types and imports;

We keep
* Module name (`GetModuleName` getter);
* Module store keys as is for now.

Will migrate the second list later. @Eengineer1 Could you please review the last change?

**Review by Eengineer1 — APPROVED** [review]
Rename & next steps look good!
Approved & thanks.

**Pull request merged**